TodoList/Sources/TodoList.Contracts/Interfaces/IListItemUrlGenerator.cs

```diff
@@ -0,0 +1,9 @@
+﻿using TodoList.Contracts.Models;
+
+namespace TodoList.Contracts.Interfaces
```

Comment:
I'm not completely against this interface being contracts, however, it will always be tightly coupled with .Api, so unless you decided to create a separate assembly for services related to the web (.Services.Web or .Services.Api, ...), I strongly suggest you move this interface right next to its implementation.